packages/components/src/checkbox-control/stories/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import CheckboxControl from '../';
+
+export default { title: 'Checkbox Control', component: CheckboxControl };
+
+export const _default = () => {
+	const [ isChecked, setState ] = useState( true );
+	return (
+		<CheckboxControl
+			label="Allow Comments"
+			help="Allow comments on post or not?"
+			checked={ isChecked }
+			onChange={ ( val ) => setState( val ) }
```

Comment:
We should show all the different combinations of `heading / label / help`.

Reply:
I added each of the variants. The heading looks off, either a CSS file isn't being loaded somewhere, or the component is not as stand-alone as expected.

Reply:
If you omit `label`, there are accessibility violations reported. We should rather promote good practices and update all examples to be violations free.

![Screen Shot 2019-10-15 at 09 18 55](https://user-images.githubusercontent.com/699132/66809041-d4d77400-ef2c-11e9-8c40-06c92c0eca3b.png)

There is another related question. Should we provide dev tooling which warns developers when they miss some required props?

Reply:
>the component is not as stand-alone as expected.

Yeah, we have this problem in a lot of components. Hopefully surfacing them like this will encourage fixing them faster.

>update all examples to be violations free.

Agreed.

>Should we provide dev tooling which warns developers when they miss some required props?

That would be great, a lot of libraries do that.